project/soulrrrrr/README.rst

```diff
@@ -0,0 +1,85 @@
+========================
+Columnar Array
+========================
+
+Basic Information
+=================
+
+https://github.com/soulrrrrr/ColumnarArray
+
+Problem to Solve
+================
+
+This problem is from https://yyc.solvcon.net/en/latest/nsd/schedule/22au_nycu/schedule.html#columnar-array.
+
+Prospective Users
+=================
+
+People originally uses Array of Struct in Python and wants to speed up.
+
+System Architecture
+===================
+
+Analyze how your system takes input, produces results, provide interface, and
```

Comment:
This paragraph does not seem to be about your architecture but a general notion.  It does not need to appear.

Reply:
Deleted this paragraph.